### DISCUSSION TITLE
Modify Style.css on allure report on jenkins

### DISCUSSION THREAD

**Discussion by automationNoob** [allure-framework/allure2#2248] [Questions & Support]
I am having an issue where execution video from native app's height and width is too big and it is overflowing from the page.
Testing with css 

.attachment__embed,.attachment__media {
    max-width: 100%;
    display: block;
    margin: auto
}

if I change the max-width to 30%, it is perfectly what I want. Is there a better way to achieve this?

Not sure how would i modify the css for allure-report as it get generated on every test and gets removed in the beginning of the build

**Comment by baev** [comment]
You can attach HTML instead of video and configure styles in HTML.

In general, I agree that there is an issue with media different media sizes. Probably we should allow users to specify additional metadata (width/height etc) per attachment.

**Reply by automationNoob** [reply]
I am using wdio-video-reporter  with allure, How would i replace video with html?

**Reply by baev** [reply]
As far as I know, this plugin has no such option.
